pubspec.yaml

```diff
@@ -1,21 +1,21 @@
 name: azure_application_insights
 description: A Dart client to integrate with Azure's Application Insights service
-version: 3.1.0
+version: 3.1.1
```

Comment:
Since the SDK major version has incremented, I think we should bump this package to version `4.0.0`?

Reply:
Yes, I would say it's a good idea. I've updated the PR.